app/models/roomrequest.py

```diff
@@ -0,0 +1,102 @@
+from .. import db
+from sqlalchemy import Column, Integer, Boolean
+import datetime
```

Comment:
Unused import I think, let's remove this!

Reply:
wait remove which one?